client/src/app/Button/Button.js

```diff
@@ -1,30 +1,55 @@
 import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
+import { motion } from 'framer-motion'
+import { clickAnimation } from '../../services/animationVariants'
+
+Button.propTypes = {
+  onClick: PropTypes.func.isRequired,
+  isRedIcon: PropTypes.bool,
+  disabled: PropTypes.bool,
+  children: PropTypes.element.isRequired,
+}
+
+export default function Button({
+  children,
+  isRedIcon,
+  disabled,
+  onClick,
+  isNoCircle,
+  isSmall,
+}) {
+  return (
+    <motion.div variants={clickAnimation} initial="rest" whileTap="pressed">
+      <ButtonStyled
+        onClick={onClick}
+        disabled={disabled}
+        isRedIcon={isRedIcon}
+        isNoCircle={isNoCircle}
+        isSmall={isSmall}
+      >
+        {children}
+      </ButtonStyled>
+    </motion.div>
+  )
+}
 
 const ButtonStyled = styled.button`
```

Comment:
I have to check this in my code as well: I remember that we should not use ``styled`` in a name?